book/src/help_vm_move.md

```diff
@@ -85,6 +85,10 @@ Options:
       --telemetry-collector-url <URL>
           URL of the OpenTelemetry collector to export tracing spans (e.g.,
           http://localhost:4317). If not set, tracing export is disabled.
+      --telemetry-service-name <NAME>
```

Comment:
This is an orthogonal issue, but probably we don't want these flags displayed prominently on short-lived subcommands like `lighthouse vm`. I think Mac is going to look into hiding a lot of the irrelevant flags by default:

- https://github.com/sigp/lighthouse/issues/7135